common/app/views/fragments/media/video.scala.html

```diff
@@ -41,7 +41,8 @@
                 data-auto-play="@player.autoPlay"
                 data-show-end-slate="@(player.showEndSlate && showEndSlate)"
                 @if(showPoster) { poster="@player.poster" }
-                data-duration="@player.video.videos.duration.toString()"
+                data-duration="@player.video.videos.formattedDuration.toString()"
```

Comment:
I think we'd want to keep `data-duration` as the raw value, or remove `data-formatted-duration` as there isn't much use in them both being the same value?